spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -87,6 +89,11 @@
 	 */
 	private String addresses;
 
+	/**
+	 * Shuffling mode for connecting host. Default mode is NONE.
```

Comment:
The default value should not be documented as it's the role of the configuration metadata to do that (look at other keys for inspiration).